### ISSUE TITLE
Whitespace Syncing

### ISSUE THREAD

**Issue by tokejepsen** [getavalon/docker#26]
**Issue**

Error when using whitespaces on project names in Kitsu.

**Reproducible**

1. Start Avalon and syncing.
2. Create "Dark Knight" project in Kitsu.

**Comment by mottosso** [comment]
Assets have the same problem.. I've resorted to [deleting whitespaces](https://github.com/getavalon/docker/blob/c96407f9668d223eeeeb702d096f422d61d78dd2/volume/sync.py#L18) during the synchronisation, but preferably there would be two names in CGWire; one with restricted characters (for e.g. file-system) and one label.

We should have a chat with Frank about it.

**Comment by tokejepsen** [comment]
Yeah, modifying the data like deleting the whitespace might cause issue when doing Avalon > CGWire.

**Comment by tokejepsen** [comment]
Started a discussion for this; https://github.com/cgwire/zou/issues/90

**Comment by tokejepsen** [comment]
In order to facilitate Avalon > CGWire syncing, we need to fix this issue.

What we could do is assume that the label is the correct name of a project/asset. This way we can more reliably map Avalon to CGWire.
Once a solution has been found on the CGWire side, we transition to it.
